### ISSUE TITLE
0xC0000008: An invalid handle was specified

### ISSUE THREAD

**Issue by torusrxxx** [x64dbg/x64dbg#3299]
When stop debugging, the 0xC0000008 exception was thrown at `MODINFO::unmapFile()`. This is because the `fileHandle` used by `StaticFileUnloadW` was set to 1 by `ModLoad` which is an invalid handle.

**Comment by mrexodia** [comment]
Yes this is on purpose, when updating TitanEngine I will fix this.
